src/f5_tts/infer/utils_infer.py

```diff
@@ -388,12 +495,23 @@ def infer_batch_process(
         ref_audio_len = audio.shape[-1] // hop_length
         if fix_duration is not None:
             duration = int(fix_duration * target_sample_rate / hop_length)
+        elif prediction_model:
+            duration_in_sec = prediction_model(audio, text_list)
+            print(duration_in_sec)
+            frame_rate = model_obj.mel_spec.sample_rate // model_obj.mel_spec.hop_length
```

Comment:
`model_obj.mel_spec.sample_rate` -> `model_obj.mel_spec.target_sample_rate`
duplicate definition of var